app/datamapper_setup.rb

```diff
@@ -0,0 +1,13 @@
+require 'data_mapper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.